src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEMethodSymbol.cs

```diff
@@ -60,10 +60,10 @@ private struct PackedFlags
             // l = isOverriddenOrHiddenMembers populated. 1 bit
             // m = isReadOnly. 1 bit.
             // n = isReadOnlyPopulated. 1 bit.
-            // 14 bits remain for future purposes.
+            // o = NullableContext. 3 bits.
```

Comment:
>o = NullableContext. 3 bits. [](start = 15, length = 28)

Is it worth caching this information? How often do we need it?

Reply:
We'll need it potentially for the return type, parameters, and type parameters if the nullable context is requested lazily in `NullableTypeDecoder`.

---
In reply to: [296841090](https://github.com/dotnet/roslyn/pull/36152#discussion_r296841090) [](ancestors = 296841090)

Reply:
> We'll need it potentially for the return type, parameters, and type parameters if the nullable context is requested lazily in NullableTypeDecoder.

Would it make sense to cache the final context rather than just the local value then?

---
In reply to: [296919021](https://github.com/dotnet/roslyn/pull/36152#discussion_r296919021) [](ancestors = 296919021,296841090)